src/clarity-angular/datagrid/datagrid-action-overflow.spec.ts

```diff
@@ -80,11 +73,18 @@ export default function(): void {
             context.detectChanges();
             expect(context.clarityDirective.open).toBe(true);
 
-            // should close when inside the overflow menu is clicked
-            let actionItem: HTMLElement = context.clarityElement.querySelector(".datagrid-action-overflow");
+            // should not close when the overflow menu container is clicked
+            let actionOverflowMenu: HTMLElement = context.clarityElement.querySelector(".datagrid-action-overflow");
+            actionOverflowMenu.click();
+            context.detectChanges();
+            expect(context.clarityDirective.open).toBe(true);
+
+            // should close when a menu item is clicked
+            let actionItem: HTMLElement = context.clarityElement.querySelector(".action-item");
             actionItem.click();
             context.detectChanges();
             expect(context.clarityDirective.open).toBe(false);
+
         });
```

Comment:
I agree that unit tests should be as simple / atomic as possible. Will update this before merging.